src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1921,6 +1930,19 @@ internal static bool CrackStringArrayInAttributeValue(out ImmutableArray<string?
             return false;
         }
 
+        private static bool ValidateMetadataArrayLength(uint length)
```

Comment:
>ValidateMetadataArrayLength [](http://example.com/codeflow?start=28&length=27)

The name is somewhat confusing, I think. The length isn't invalid, it indicates null value.